### ISSUE TITLE
CONECT/LINK line-length bug fix #2 for PDB Parser

### ISSUE THREAD

**Issue by Sheraf-DNG** [biojava/biojava#760]
Hi All, 

I'm working with custom PDB Files and I'm getting StringIndexOutOufBondsException from the PDBParser.
I've seen you rencently added this line in the PDBParser  (conect_helper (String line,int start,int end) method) : 

if  line.length() < end) return null;

It'b be nice if to check the line.length() do not exeed the start arg, should be fixed by changing it to this (tested):

if (line.length() < start || line.length() < end) return null;

Thanks for the good work!

**Comment by josemduarte** [comment]
Good catch, could you provide a pull request so that we can review it and merge if everything looks fine? It's difficult to follow exactly what your changes are without the code context.

See here for instructions on how to do a pull request: https://help.github.com/articles/creating-a-pull-request/

**Comment by r614** [comment]
Hey! I fixed the bug on my local branch but I am unable to push the branch to upstream to make a PR. I require permissions!

**Comment by lafita** [comment]
Hi @r614, we do not allow directly pushing changes to the main repository. You should instead fork the biojava repository to your account, push the changes there, and then open a new PR. You can find more info in the [contributing instructions](https://github.com/biojava/biojava/blob/master/CONTRIBUTING.md).

**Comment by r614** [comment]
Yikes, that's my bad. Just added a PR.
